src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/MsiInstallerStrings_de.wxl

```diff
@@ -1,5 +1,5 @@
 <?xml version = '1.0' encoding = 'utf-8'?>
-<WixLocalization Culture="de-de" xmlns="http://schemas.microsoft.com/wix/2006/localization" Codepage="1252">

```

Comment:
Please revert the value of the `Culture` attribute.

Reply:
I resolved this for the German, Japanese, and Chinese files in the latest commit.